tr_sys/tr_sys/settings.py

```diff
@@ -53,6 +53,13 @@
     'tr_kp_molecular.molecular_app.AppConfig',
     'tr_kp_cam.cam_app.AppConfig',
     'tr_kp_textmining.textmining_app.AppConfig',
+    'tr_ara_aragorn_exp.aragorn_exp_app.AppConfig',
+    'django.contrib.admin',
```

Comment:
This is the result of a bad merge

Reply:
Ah, I did resolve a conflict there, but I'm not quite sure what you changed?